graphene_django/settings.py

```diff
@@ -41,6 +41,10 @@
     "DJANGO_CHOICE_FIELD_ENUM_CUSTOM_NAME": None,
     # Use a separate path for handling subscriptions.
     "SUBSCRIPTION_PATH": None,
+    # Set to True to enable GraphiQL headers editor tab
+    # This sets headerEditorEnabled GraphiQL option, for details go to
+    # https://github.com/graphql/graphiql/tree/main/packages/graphiql#options
+    "GRAPHIQL_HEADER_EDITOR_ENABLED": False,
```

Comment:
Just curious, why not always have it enabled? I think the majority use case for GraphiQL in Django is going to be development, no? Having the header panel always enabled seems like it would be fine. If someone _really_ wants to disable it, they can copy the template and configure GraphiQL themselves.

If we decide to keep it, you'll also need to update `views.py` to pass through the configuration option [like we do with `subscription_path`](https://github.com/graphql-python/graphene-django/blob/master/graphene_django/views.py#L168-L169) as well as the [template where it gets injected](https://github.com/graphql-python/graphene-django/blob/master/graphene_django/templates/graphene/graphiql.html#L43-L49).

Reply:
I did not want to enable it by default to not change current behaviour. But I have no objections to make it `True` by default if that is desired.

I will have a look at `views.py` and template later today, thanks for info.

Reply:
@radekwlsk I love the focus on stability and definitely appreciate the forethought!

Given that GraphiQL is provided by `graphene-django` as a convenient developer tool outside of the critical path, I don't think it has the same requirements for interface stability as the rest of the library. We upgraded GraphiQL to the latest version very recently in 2.12 which changed the interface quite a bit, and had I known about this feature at the time, I would have just turned it on in that PR.

@jkimbo @mvanlonden any feelings on this? Turning it on by default would make this a very small JS-only PR and one less setting to care about.

Reply:
@eabruzzese I've injected the setting to template. With enabling headers tab by default I will let a little bit more discussion to happen first.

Reply:
I'm fine with enabling the setting by default since it's purely an enhancement and it shouldn't break anything.